specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json

```diff
@@ -0,0 +1,4575 @@
+﻿{
```

Comment:
Discussed offline. As this is an aggregation of existing PRs which have already been reviewed, please provide a lit of changes that haven't been reviewed and I will look through these.

Reply:
1.	Renamed some of the example files to reflect the actual API names in the swagger (e.g. “ProjectPermissionsList.json” -> “InvoiceSectionPermissionsList.json”).
2.	Changed the variable names in the paths to better reflect ARM naming conventions, e.g. “…/billingAccounts/{billingAccountId}” is now “…/billingAccounts/{billingAccountName}”.
3.	Addressed some of the warnings/errors generated by the current OpenAPI hub validation tools:
a.	property names using camelCasing instead of PascalCasing
b.	Ensured all async operations are marked as sync (e.g. InvoiceSections_Create was missing "x-ms-long-running-operation": true)
c.	Added 200 responses to all async operations
d.	Changed the operationId in several cases (some operationIds had multiple underscores in them, others were inconsistent with other names)
4.	Removed unused definition “BillingContact”

Reply:
The way this was done makes it incredibly difficult to review properly. When working on large features, you should work with the repo maintainers to create a separate branch where you do your incremental work. Then, there'd be one PR into master.

---
In reply to: [267554449](https://github.com/Azure/azure-rest-api-specs/pull/5423#discussion_r267554449) [](ancestors = 267554449)

Reply:
Based on the summary of changes on top of the aggregation of signed off PRs, I'll sign off from the ARM side.

---
In reply to: [268843442](https://github.com/Azure/azure-rest-api-specs/pull/5423#discussion_r268843442) [](ancestors = 268843442,267554449)